app/assets/stylesheets/new_board/_screen.scss

```diff
@@ -1,3 +1,41 @@
 .full-heigth {
     height:100%;
 }
+
+.markdown-wrapper{
+  background: $white;
+  border-left: 4px solid $lightgrey-17;
+  margin-top: 5px;
+  padding: 10px;
+
+  &__text-rigth{
+    float: right;
+    vertical-align: top;
+    color: $darkgrey-8;
+    font-size: 0.75em;
+  }
+}
+
+.popover {
+  &__content {
+    padding: 5px 5px 0 5px;
+    word-wrap: break-word;
+
+    &__title {
+      font-size: 13px;
```

Comment:
Should `font-size` values always use `px` or `em`? Is using both intended?

Reply:
It is intended, we are copying the css from the v1.